tests/scripts/kubeadm.sh

```diff
@@ -0,0 +1,64 @@
+#!/bin/bash +e
+
+scriptdir="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
+
+tarname=image.tar
+tarfile=${WORK_DIR}/tests/${tarname}
+
+export KUBE_VERSION=${KUBE_VERSION:-"v1.6.7"}
+
+
+kubeadm_init() {
+
+    sudo kubeadm init --skip-preflight-checks
+
+    sudo cp /etc/kubernetes/admin.conf $HOME/
+    sudo chown $(id -u):$(id -g) $HOME/admin.conf
+    export KUBECONFIG=$HOME/admin.conf
+
+    kubectl taint nodes --all node-role.kubernetes.io/master-
+    kubectl apply -f https://git.io/weave-kube-1.6
```

Comment:
does the weave version depend on the k8s version? ie. if we run k8s1.7 do we need weave 1.7?